src/source.js

```diff
@@ -0,0 +1,50 @@
+export function getData(category) {
```

Comment:
Get data and get links same methods, so repeatable code, remove one, made single one to could call with param and without param. But I think you never need to call without category parameter.